lms/djangoapps/grades/tests/test_models.py

```diff
@@ -32,7 +32,7 @@ def setUp(self):
         )
 
     def test_empty_block_record_set(self):
-        empty_json = '{0}"blocks":[],"course_key":"{1}"{2}'.format('{', unicode(self.course_key), '}')
+        empty_json = u'{0}"blocks":[],"course_key":"{1}","version":1{2}'.format(u'{', unicode(self.course_key), u'}')
```

Comment:
Same nit as before - could we use `BLOCK_RECORD_VERSION` instead of a hardcoded `1` in these tests?